ext/GeometryOpsDimensionalDataExt/GeometryOpsDimensionalDataExt.jl

```diff
@@ -0,0 +1,20 @@
+module GeometryOpsDimensionalDataExt
```

Comment:
Should we close the PR that does this in GeometryBasics?

Reply:
No I meant st have added that by mistake lol